src/sbvr-api/sbvr-utils.coffee

```diff
@@ -540,14 +541,34 @@ exports.runRule = do ->
 		.nodeify(callback)
 
 exports.PinejsClient =
-	class PinejsClient extends PinejsClientCore(_, Promise)
-		_request: ({ method, url, body, tx, req, custom }) ->
+	class PinejsClient extends PinejsClientCore(_, Promise, mimemessage)
+		compileBatch: (data) -> _.map(data, batchify)
```

Comment:
Doesn't this need to run the requests via pinejsClient.compile like in https://github.com/resin-io/pinejs-client-js/pull/34/files#diff-7211713139c36b3a80c048e901477db2R488 ?